design-system/materials/internals/definition.yaml

```diff
@@ -946,6 +984,13 @@ decisionGroupsByTheme:
         height-for-input:
           choice: '40px'
 
+    widths:
+      label: Width
+      prefix: width
+      decisions:
+        width-for-avatar-as-medium:
+          choice: '40px'
```

Comment:
Should we include the other sizes as well?

Reply:
I also thought about this and my take here is that it depends on what we want to do with design tokens after we finish the migration: do we want to populate the `definition.yaml` file with all the css values to have a centralized source of truth so all our components should be tokenized and they wouldn't have any css value but just tokens references? If that's so, I guess it would make sense to bring the other values. Otherwise, I think we would be removing this token after the migration and just updating the values in the `avatar.tsx` file.

This is one of the questions I think we need to discuss about after the theme migration to decide how we want to manage styles in `ui-kit` for the future.

Reply:
Should I merge this PR or should I wait until we figure this out?

Reply:
Let's merge this and continue the discussion separately.